package.json

```diff
@@ -27,6 +27,7 @@
     "mocha": "^10.0.0"
   },
   "dependencies": {
-    "axios": "^0.27.2"
+    "axios": "^0.27.2",
+    "mochawesome": "^7.1.3"
```

Comment:
As it is a Dev Dependency, it shouldn't be here
Mochawesome must be in the devDependencies

Reply:
```suggestion
    `npm uninstall mochawesome`
    `npm install mochawesome -D`
```